src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -51,6 +51,7 @@
     <Compile Include="File\ReadWriteAllLinesAsync.cs" />
     <Compile Include="File\ReadWriteAllBytesAsync.cs" />
     <Compile Include="File\ReadWriteAllTextAsync.cs" />
+    <Compile Include="FileStream\ReadWriteSpan.netcoreapp.cs" />
```

Comment:
These changes will presuambly light up in Project N in due course at which point we might want to add a UAP configuration to these tests and include this file.

In general we don't really have a good process to make sure we enable netcoreapp tests for UAP when the changes reach there.